test/Orleans.Transactions.Tests/Memory/MemoryTransactionsFixture.cs

```diff
@@ -27,7 +25,7 @@ public ISiloHostBuilder CreateSiloBuilder(string siloName, ClusterConfiguration
                     .ConfigureSiloName(siloName)
                     .UseConfiguration(clusterConfiguration)
                     .ConfigureLogging(builder => TestingUtils.ConfigureDefaultLoggingBuilder(builder, TestingUtils.CreateTraceFileName(siloName, clusterConfiguration.Globals.ClusterId)))
-                    .UseInClusterTransactionManager(new TransactionsConfiguration())
+                    .UseInClusterTransactionManager(options => { })
```

Comment:
You seem to be using defaults very often... If this is the most likely scenario for end-users, consider and overload that doesn't take any parameters (or an optional parameter that by default is null). If this would be common just in tests, then ignore the suggestion

Reply:
Defaults should be common for this.  Will had overload or something.